lsp-ui-doc.el

```diff
@@ -702,12 +702,18 @@ FN is the function to call on click."
 (defvar-local lsp-ui-doc--inline-width nil)
 
 (defun lsp-ui-doc--inline-window-width nil
+  "Calculate inner padding for the child frame showing the documentation"
+  (if (not (bound-and-true-p lsp-ui-doc-use-childframe))
+      (setq additional-width-spacing 8)
```

Comment:
the value of `8` here is a complete guess of what the minimum good value to fix the problem is. Same reasoning for the default value of `1` that lsp-ui-doc is already setting as an alternate value.

Reply:
Make sense to me.